src/Microsoft.AspNetCore.Sockets.Client/SendMessage.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Sockets.Client
+{
+    public struct SendMessage
```

Comment:
I couldn't come up with a better name for the `SendMessage` - I would be more than happy to change it to something better. 

The Message is called just `Message` because it is shared with the server where the channels are symmetric (i.e. objects of the same type are sent both directions).